pkg/verifier/notation/notation.go

```diff
@@ -57,8 +68,8 @@ type NotationPluginVerifierConfig struct { //nolint:revive // ignore linter to h
 
 	// VerificationCerts is array of directories containing certificates.
 	VerificationCerts []string `json:"verificationCerts"`
-	// VerificationCerts is map defining which keyvault certificates belong to which trust store
-	VerificationCertStores map[string][]string `json:"verificationCertStores"`
+	// VerificationCerts is map defining which keyvault certificates belong to which trust store name and its trust store type.
```

Comment:
it would be good to add an example explaining how store name/store type maps to the value.

Reply:
I would add a sample here